assayist/processor/base.py

```diff
@@ -32,16 +68,16 @@ def main(self):
 
         :param str input_dir: The directory in which to find the files.
         """
-        neomodel.db.set_connection(config.DATABASE_URL)
+        db.set_connection(config.DATABASE_URL)
         # run the analyzer in a transaction
-        neomodel.db.begin()
+        db.begin()
         try:
             self.run()
             log.debug('Analyzer completed successfully, committing.')
-            neomodel.db.commit()
+            db.commit()
         except Exception as e:
```

Comment:
I know this is unrelated but `e` is unused here and the comment refers to a non-existing argument.

Reply:
Ah, arg got moved to __init__.